src/main/kotlin/nebula/plugin/compile/JavaCrossCompilePlugin.kt

```diff
@@ -55,16 +52,13 @@ class JavaCrossCompilePlugin : Plugin<Project> {
                         }
                     }
                 }
-                withType(GroovyCompile::class.java) {
```

Comment:
I tested this and setting bootstrap classpath for groovy compilation has no effect.

Reply:
Meaning the feature is already not working. I was able to produce Groovy classes with 1.8 bytecode but referencing APIs from 11.